great_expectations/rule_based_profiler/data_assistant/data_assistant_dispatcher.py

```diff
@@ -61,3 +62,54 @@ def __getattr__(self, name) -> DataAssistantRunner:
             ] = data_assistant_runner
 
         return data_assistant_runner
+
+    @staticmethod
+    def register_data_assistant(
+        data_assistant: Type[DataAssistant],  # noqa: F821
+    ) -> None:
+        """
+        This method executes "run()" of effective "RuleBasedProfiler" and fills "DataAssistantResult" object with outputs.
+
+        Args:
+            data_assistant: "DataAssistant" class to be registered
+        """
+        data_assistant_type = data_assistant.data_assistant_type
+        registered_data_assistants: Dict[
```

Comment:
do we need this type hint? can it not be very directly inferred?

Reply:
it can for sure but we've been leaning towards being very explicit with everything. `mypy` and other static analysis tools don't require the additional typing but per discussion with Alex, they don't hurt.

I'm on the fence and could go either way.